### DISCUSSION TITLE
Callback function when Menu Activated

### DISCUSSION THREAD

**Discussion by markmccoid** [enesozturk/react-native-hold-menu#12] [Ideas]
I would like to be able to add a haptic when the menu is opened.

I think the easiest way would be to allow for a callback function to be passed to the HoldItem component and have it called when the menu open is triggered.

Thanks,

**Comment by enesozturk** [comment]
Hi @markmccoid, we already added haptic feedback support for hold menu, you can check the details here: https://github.com/enesozturk/react-native-hold-menu/pull/5

For now, did not released new version because hold menu has an important issue: https://github.com/software-mansion/react-native-reanimated/issues/1758. I will release haptic support and other features as soon as it is solved.

**Reply by markmccoid** [reply]
After upgrading to Expo SDK 41, I believe I'm hitting the error you mention above.  

I noticed that in the issue at Software Mansion you mentioned that you were not getting the error.  Was that a change in the code or something else?

Expo 41 SDK is using reanimated ~2.1.0.

Thanks!

**Reply by enesozturk** [reply]
Hi @markmccoid, Just yesterday I got the same problem. Something wrong with Expo 41. Reanimated 2.1.0 is working on bare React Native setup but not working with Expo. Will consider and let the SM know. Thanks for letting me know 👍🏽
